### DISCUSSION TITLE
Usage with Lit and Storybook?

### DISCUSSION THREAD

**Discussion by akarpov91** [facebook/stylex#564] [Q&A]
Hi, I've been doing some research on how to integrate Stylex with Lit and Storybook for a component library, but haven't been able to find much info. 

Can someone point to some documentation or github examples that may help with my intent?

Thanks in advance.

**Comment by mattstyles** [comment]
I don't know about Lit but Storybook allows you to customise your build.
You can add the stylex transform into that build customisation.

**Comment by nmn** [comment]
Using StyleX with Lit is possible if you opt out of shadow DOM. Shadow DOM is inherently incompatible with atomic CSS and will involve loading the same CSS for every component instance, which is not ideal and not performant in some of the browsers.

Outside of that, it all depends on what bundler you're using. There have been questions about Storybook before. I don't have a lot of directly experience with it, so I'll defer to the answers from the community.
